product_development.md

```diff
@@ -0,0 +1,96 @@
+
+### Bugs
+
+Bugs are owned by the engineering team as they affect the platform's stability.
+
+Bugs must have some kind of urgency. They are not prioritized by product, but triaged by engineering, based on what bug will kill our business first. Triage is handled by a designated senior engineer. Bugs are assigned to an engineer at all times but can be reassigned.
+
+A bug ticket should have an average resolution rate of less than a week.
+
+Bugs may be closed or converted into a story, if there is no longer a high urgency and no impact on the stability of the platform. Updates on bug tickets, especially closure without resolution, need to be communicated with product proactively.
+
+### Tech debt
+
+Engineering leads are responsible for the stability of the platform and the development velocity, and consequently for when and what technical debt needs to be handled.
+
+Work on technical debt must always be proactively communicated with the product team and documented in a ticket. Tech debt tickets must be pulled into the backlog in a refinement meeting, as they bind resources and take up significant space in the engineering backlog.
+
+### Platform announcement Slack channel
+
+Product, Design, Engineering, and Content team members post every change to go live on the platform into the Slack channel [#platform-announcements](https://voiio.slack.com/archives/C02BXK6R16V). This channel serves as a changelog for all employees.
+
+## Meetings & Collaboration
+
+### Agile principles
+
+* Make work visible
+* Meaningful & actionable conversations
+* Decisions are made, where the work is done
+* Regularly reflect on how we work together
+
+### Stakeholder Meetings
+
+Stakeholder meetings are held weekly to discuss the product roadmap, upcoming features, and any blockers that need to be resolved or questions that need to be answered. It is the best path to discuss new ideas and to align on the product's direction. The product manager will relate work to the OKR progress and stakeholders will come prepared with issues/ideas and data, adding their topics to this document: [Stakeholder Meeting Platform](https://docs.google.com/document/d/1uhX68whx9T56unUlEUdyjUdq_dOJpHwO7Ycv892hqA8/edit)
+
+The meeting is led by the Product Manager and attended by key stakeholders and an engineering lead.
+
+### Refinement Meetings
+
+Refinement meetings are held weekly and are the handover point from the Product Team to the Engineering Team. The product manager presents problems, opportunities, and priorities to the Engineering Team. Engineers ask questions and either pull the issue into the backlog or push it back to the product team for further refinement.
+
+The meeting is led by the product manager and attended by the product and engineering team.
```

Comment:
Sometimes teams are capitalized (Product Team) and sometimes are not (product team). Let's align on the style. :)

Reply:
AI ;) Should be all lowcaps